tests/unit/orchestrator/test_models.py

```
@@ -0,0 +1,83 @@
+
+
+class TestRunResult:
+    """Tests for RunResult data model."""
+
+    def test_create_successful_run_result(self):
+        """Test creating a successful RunResult."""
+        result = RunResult(
+            label="run_0001",
+            success=True,
+            summary_metrics={"ttft_avg": 100.0, "tpot_avg": 10.0},
+            artifacts_path=Path("/tmp/run_0001")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/run_0001"

(S108)